src/daily_output.ts

```diff
@@ -203,6 +203,11 @@ export class DailyOutput {
 					this.data.ranks.total.groupEffortRewards.push({ address, rewards, rankPosition: 0 });
 				}
 			});
+			this.data.ranks.total.groupEffortRewards = this.data.ranks.total.groupEffortRewards
+				.sort(({ address: address_1 }, { address: address_2 }) =>
+					weeklyTiebreakerSortFactory(this.data.wallets)(address_1, address_2)
```

Comment:
We want a `total` tiebreaker, not weekly

Reply:
THAT'S WHY IT'S IMPORTANT TO HAVE MEANINGFUL NAMES! I ended up renaming that because I realized I was only using `weekly` in the method implementation. 
Thanks! Nice catch! :sunglasses:

Reply:
There it is: 44018fc2b1cf538aaf942a012a729c190796c077